### PR TITLE
LPS-62410 SF - No need of global availability anymore.

### PR DESCRIPTION
**Pull request by brandizzi**
I've done a fair amount of refactoring here. Let's see what CI will report.

cc @natocesarrego

### CODE REVIEW

**Comment by vicnate5** [comment]
ci failures are not related to this pull

**Comment by natocesarrego** [comment]
Nice @vicnate5! Thanks.

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/interval_selector_scheduler_event_link.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+			{
+				ATTRS: {
+					intervalSelector: {
+					},
+
+					schedulerEvent: {
+					}
+				},
+
+				AUGMENTS: [Liferay.PortletBase],
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/interval_selector_scheduler_event_link.js]: @brandizzi Is the PortletBase augmentation necessary? I can't see where you are using it's methods.

Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/interval_selector_scheduler_event_link.js]: Indeed, there is no need of that. I submitted a new PR without it, and with a minor fix as well. See https://github.com/brunobasto/liferay-portal/pull/666.

Thanks for your time!

**Pull request closed without merging**